release/__init__.py

```diff
@@ -267,23 +267,36 @@ def get_gen_package_artifact(package_id_str):
         'local_path': package_filename}
 
 
-def make_bootstrap_artifacts(bootstrap_id, variant_name, artifact_prefix):
+def make_bootstrap_artifacts(bootstrap_id, package_ids, variant_name, artifact_prefix):
     bootstrap_filename = "{}.bootstrap.tar.xz".format(bootstrap_id)
+    bootstrap_filename = "{}.bootstrap.tar.xz".format(bootstrap_id)
+    active_filename = "{}.active.json".format(bootstrap_id)
+    active_local_path = artifact_prefix + '/bootstrap/' + active_filename
+    latest_filename = "{}bootstrap.latest".format(pkgpanda.util.variant_prefix(variant_name))
+    latest_complete_filename = "{}complete.latest.json".format(pkgpanda.util.variant_prefix(variant_name))
+
+    # Assert that the bootstrap active packages are in the package list.
+    with open(active_local_path) as f:
+        missing_packages = set(json.load(f)) - set(package_ids)
```

Comment:
1. The set of packages activated in the bootstrap isn't published/returned as build metadata, only the variant's bootstrap ID and complete set of packages. We could return that metadata from a build, but it would be irrelevant outside of validation.
1. 👍 
1. 👍